src/quicktype-core/language/CSharp.ts

```diff
@@ -507,6 +507,7 @@ export const newtonsoftCSharpOptions = Object.assign({}, cSharpOptions, {
         ["attributes-only", { helpers: false, attributes: true }],
         ["just-types", { helpers: false, attributes: false }]
     ]),
+    superclass: new EnumOption("superclass", "Superclass", [["EntityData", "EntityData"], ["none", undefined]], "none"),
```

Comment:
Oh, one more thing: Please add the option `"secondary"` at the end.  Then it'll show up in the "Other" tab in the UI.